.github/workflows/install/action.yml

```diff
@@ -38,7 +38,7 @@ runs:
 
     - name: Cache dependencies
       id: cache-dependencies
-      uses: actions/cache@v2
+      uses: actions/cache@v3
```

Comment:
At first glance of the [release notes](https://github.com/actions/cache/releases/tag/v3.0.0) for `actions/cache@v3` it looks like they changed the minimum supported node version from `12` to `16`, and we're using `14.15.0`, but I think the notes are talking about the minimum node version on the runner itself. This isn't a problem with `ubuntu-latest` since it installs `16.16` by [default](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#language-and-runtime). Not sure about our self-hosted runners, but it seems like it's working fine in `content-build`